lib/apiservers/engine/backends/container.go

```diff
@@ -1431,3 +1473,72 @@ func copyEscapable(dst io.Writer, src io.ReadCloser, keys []byte) (written int64
 func clientFriendlyContainerName(name string) string {
 	return fmt.Sprintf("/%s", name)
 }
+
+func processVolumeParam(volString string) (volumeFields, error) {
```

Comment:
I was thinking about that actually. I have been unhappy with how large this file is as well. I don't like having to memorize line numbers to navigate a file. Since these are part of the engine I would rather move all container files into the container directory of the repo as well. Perhaps that can be a cleanup for another time though.